package.json

```diff
@@ -2,9 +2,11 @@
   "name": "typist",
   "version": "0.5.0",
   "private": true,
+  "main": "public/electron.js",
```

Comment:
Is this needed?

Reply:
Electron read the `package.json` and look for `main` value, is the entrypoint for Electron